config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -112,7 +112,7 @@ periodics:
       - --deployment=node
       - --gcp-project-type=node-e2e-project
       - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/benchmark-config.yaml --test-suite=conformance
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml --test-suite=conformance
```

Comment:
I saw in benchmark-config.yaml, we tried to run different tests against the same image. Is it designed differently to run a test-suite on the same image? Wonder whether it is intentional if it do "up, run test1, down", "up, run test2, down"  or "up, run test1, run test2, run test3, down"

Reply:
I don't understand. 

benchmark config lists single test per image.

Reply:
What I mean for benchmark-config.yaml , each test will be running:
create cluster, run test1, teardown cluster;
create cluster, runt test2, teardown cluster;

For image-config.yaml, it will run in the following:
create cluster, run test1,  run test2, teardown cluster;

If it is above case,  benchmark-config.yaml  is preferred to avoid interfering between tests.

Reply:
Yes, exactly.

This is not running benchmarks.

Reply:
Cool, this is a good fix.